maquetado/scss/styles.scss

```diff
@@ -11,9 +11,11 @@
 }
 
 .book-cover {
-  height: 368px;
+  height: 100%;
+  max-width: 380px;
```

Comment:
This is not necessary when it comes to height. You can just leave `height: 380px`.